students/13th/hamjunho/js/main.js

```diff
@@ -0,0 +1,289 @@
+      nickname: 'page0blue',
+      message: '귀찮네'
+    },
+    {
+      icon: 'https://ww.namu.la/s/aeca2e14dbb78281beabffe6d5a8b1a84233da4aa2d7b857f11baa8530908b4faf2dae5ce55389c73821b2235cfa3c4d9744bb30d3edc6efda4a446164481c7883a1601fe728564b751b30bb4e7b5fec0c9b9560abda620ddf868d89baf37357aee9213d8393be29d72c5d821151ba8b',
+      nickname: 'EasyGold',
+      message: '이지금'
+    },
+    {
+      icon: 'https://ww.namu.la/s/4e68d2da7f9dc6ec1251daa195275de1a481dbef92a1f00a627dc5f7bdc1aeb8dcfb3dee162cb279e0b28e785e141a99af2559dad5cebfcaa63bad4013cf6edc21d55d1cad4d4fac317161f8f1801c916355ce3711c1a9e4121d7dfc49034ecae954781309df50a86f607b41316f8e26',
+      nickname: 'moonLight',
+      message: '문별'
+    },
+    {
+      icon: 'https://w.namu.la/s/b99033a5fa0125d8bed88db1287d8d6e44ecbaa56f51941c0adc6a9dc5179e6651aeee3e3c7f95bc50d9eb4ac432b731f8bdf6bf941c87d1133b9586e5af736ba7aa52c31a153184c1850b642ae9a52548d876c9ece5e433187a6abfb954741b34400921306aa3f85b9657054c813377',
+      nickname: 'jiminstart',
+      message: '정정'
+    },
+  ]
+}
+
+const dummyPost = {
+  user: {
+    nickname: 'page0blue',
+    icon: 'https://scontent-gmp1-1.cdninstagram.com/v/t51.2885-19/s320x320/50959409_809292579425612_573364557624377344_n.jpg?_nc_ht=scontent-gmp1-1.cdninstagram.com&_nc_ohc=B9rbwtW2wU8AX-RyzAc&oh=a001eb7f445dd801f901f276a9fdde5e&oe=5F91AAAC'
+  },
+  postId: 0,
+  images: {
+    src: 'https://file.mk.co.kr/meet/neds/2020/04/image_readtop_2020_435990_15879677954178860.png'
+  },
+  contents: '날씨가 너무 좋아',
+  comments: [
+    {
+      nickname: 'kwe1s',
+      contents: '첫번째 댓글',
+      liked: true,
+    },
+    {
+      nickname: 'kwe2s',
+      contents: '두번째 댓글',
+      liked: false,
+    }
+  ],
+  liked: 100,
+}
+/// dummy 부분 ----
+
+const _map = (f, iter) => { let res = []; for (const a of iter) { res.push(f(a)); } return res; }
+
+
+const body = document.querySelector('body');
+
+const postNickname = document.querySelectorAll('.user_name');
+_map((v) => v.textContent = dummyPost.user.nickname, postNickname)
+
+const profileImage = document.querySelectorAll('.my_profile_image');
+// profileImage.map((v) => v.classList.add('clipped'))
+_map((v) => {
+  v.setAttribute("src", dummyPost.user.icon)
+  v.classList.add('clipped')
+}
+, profileImage)
+
+const getFriendImage = document.querySelectorAll('.get_friend_image');
+_map((v) => v.setAttribute("src", 'https://image.bugsm.co.kr/artist/images/1000/800491/80049126.jpg'), getFriendImage)
+
+const postImage = document.querySelector('.feed_image img');
+postImage.setAttribute("src", dummyPost.images.src)
+
+const PostMessage = document.querySelector('.feed_message');
+PostMessage.textContent = dummyPost.contents;
+
+const likeYouMessage = document.querySelector('.like_you_message');
+likeYouMessage.innerHTML = `<span class="bold">null</span> 님 <span 
+class="bold">외 ${dummyPost.liked}명</span>이 좋아합니다`;
```

Comment:
```suggestion
likeYouMessage.innerHTML = `
  <span class="bold">null</span> 님 
  <span class="bold">외 ${dummyPost.liked}명</span>이 좋아합니다
`;
```

- 줄바꿈만 잘해줘도 가독성이 좋아집니다!